packages/docs/components/layout/icons/CollapseIcon/index.vue

```diff
@@ -17,7 +17,7 @@
 </script>
 
 <template>
-  <component
+  <Component
```

Comment:
`Component` with capital `C`

Reply:
I saw `component` was used with small first letter in vue docs (https://vuejs.org/api/built-in-special-elements.html#component) I'd prefer it be in lowercase, cuz it is built-in component in vue js, not custom one. Notice that this doesn't work for Transition (see https://vuejs.org/guide/built-ins/transition-group.html#transitiongroup) :) 


> `<component>`, `<slot>` and `<template>` are component-like features and part of the template syntax. They are not true components and are compiled away during template compilation. As such, they are conventionally written with lowercase in templates.